vis/emissions.R

```diff
@@ -49,13 +49,12 @@ individual_emissions$category = factor(
 individual_emissions <- individual_emissions %>% 
   mutate_at(c(3:9), as.numeric)
 
-##########################
-##plot1 
-##########################
+##################################
+##plot1  climate change baseline##
+##################################
 df = individual_emissions %>%
   group_by(`Constellation`, category) %>%
-  summarize(mean = mean(`Climate Change - Global Warming Potential 100a`)
-  )
+  summarize(mean = `Climate Change - Global Warming Potential 100a`)
```

Comment:
This isn't a mean value though - this is massively confusing to external reviewers. If something isn't a mean, do not call it a mean. If you need to call it something else, use "value". Or some other generic term.